wp-content/plugins/ig-revisions/plugin.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Plugin Name: Integreat Page Revisions
+ * Description: Set a specific page revision for delivery via API
+ * Version: 1.0
+ * Author: Sven Seeberg
+ * Author URI: https://github.com/Integreat
+ * License: MIT
+ * Text Domain: ig-revisions
+ */
+
+function ig_revisions_metabox( $post ) {
+	$children = wp_get_post_revisions( $post_id );
```

Comment:
`$post_id` is undefined at this point, but wordpress falls back to the correct post anyway, so this has no consequences except throwing a php notice.